### PR TITLE
feat(#58): page de paiement ( design & refonte ) 

### PR DESCRIPTION
**Pull request by CinquinAndy**
#58 

- [x] Desktop view 
- [x] Tablet view
- [x] Mobile view

### CODE REVIEW

**Review by CinquinAndy — COMMENTED** [review]
youhou

File: src/routes/checkout/+page.svelte [hunk 1]
```diff
@@ -52,8 +52,8 @@
 			elements,
 			redirect: 'if_required',
 		})
-		// log results, for debugging
-		console.log({ result })
+		// // log results, for debugging
+		// console.log({ result })
```
Comment (LightInn) [inline src/routes/checkout/+page.svelte:56]: deux commentaires inutile ?

File: src/utils/const.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
 export const api_origin = 'https://artriste-api.beta.andy-cinquin.fr/api/files/'
 export const product_infos = [
+	'Le tableau est unique et vous en serez le seul et unique propriétaire.',
+	"Le tableau est envoyé avec un certificat d'authenticité.",
```
Comment (LightInn) [inline src/utils/const.ts:4]: quote et guillement melanger

**Review by LightInn — APPROVED** [review]
❤️

**Pull request merged**